androguard/core/bytecodes/apk.py

```diff
@@ -804,39 +796,63 @@ def _format_value(self, value):
                     value = self.package + "." + value
         return value
 
-    def get_element(self, tag_name, attribute, **attribute_filter):
```

Comment:
same here, I would leave it in the code...

maybe mark both in the docstring with 
```
.. deprecated:: 3.3.5
    use :func:`whatever` instead
```

If we leave it in, we could merge it for 3.3.5, otherwise we should wait for 3.4.0 as it would break the API.